components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -179,6 +179,7 @@ public class SCIMUserManager implements UserManager {
     private static final String DISPLAY_ORDER_PROPERTY = "displayOrder";
     private static final String REGULAR_EXPRESSION_PROPERTY = "regEx";
     private static final String EXCLUDED_USER_STORES_PROPERTY = "excludedUserStores";
+    private static final String SHARED_PROFILE_VALUE_RESOLVING_METHOD_PROPERTY = "sharedProfileValueResolvingMethod";
```

Comment:
Don't we have a public constant defined for this in any other place?

Reply:
no.

ClaimConstants.SHARED_PROFILE_VALUE_RESOLVING_METHOD has the same value but starting `S` is captial.

When returning the properties in the schema all property names were started with simple letters.
So following the same pattern as other properties